Mathlib/CategoryTheory/Monoidal/Functor.lean

```diff
@@ -1032,6 +1032,11 @@ lemma functor_map_μ_inverse_comp_counitIso_hom_app_tensor (X Y : D) :
       δ e.functor _ _ ≫ (e.counitIso.hom.app X ⊗ e.counitIso.hom.app Y) :=
   e.toAdjunction.map_μ_comp_counit_app_tensor X Y
 
+set_option linter.style.nameCheck false in
+@[deprecated (since := "2024-11-30")]
+alias unitIso_hom_app_tensor_comp_inverse_map_δ_functor__ :=
+unitIso_hom_app_tensor_comp_inverse_map_δ_functor
+
```

Comment:
```suggestion
```

Reply:
`unitIso_hom_app_tensor_comp_inverse_map_δ_functor__` was deleted in #17956 without leaving a deprecation note.